nouvelles-perspectives-en-sciences-sociales.csl

```diff
@@ -24,10 +26,10 @@
       <term name="translator" form="verb-short">trad. par</term>
       <term name="translator" form="short">traduction</term>
       <term name="interviewer" form="verb">entretien réalisé par</term>
-      <term name="in">dans </term>
+      <term name="in">dans&#160;</term>
```

Comment:
&#160;

Reply:
This modification will not affect our proposition ? We need a bibliography for our journal as the sample citations that csl-bot sent us. The citations generated and based on our proposed changes.

Reply:
@lsaadaoui, no, `&#160;` is just a different (escaped) XML representation of a non-breaking space. We just prefer to see the special character codes as visually these non-breaking spaces are indistinguishable from regular spaces. It doesn't change how the CSL style behaves.

Reply:
Thanks for the clarifications